lib/tasks/limber.rake

```diff
@@ -223,10 +223,6 @@ namespace :limber do
         )
       end
 
-      lcmb_catalogue =
```

Comment:
Does this need to be replaced with anything else ?

Reply:
I think it's all handled by the record loader additions above